Makefile

```diff
@@ -51,8 +51,12 @@ sd-svs-disp: prep-salt ## Provisions SD Submission Viewing VM
 	sudo qubesctl top.enable sd-svs-disp
 	sudo qubesctl --targets sd-svs-disp state.highstate
 
+clean-salt: assert-dom0 ## Purges SD Salt configuration from dom0
+	sudo rm -rf /srv/salt/sd
+	sudo find /srv/salt -maxdepth 1 -type f -iname 'sd*' -delete
+	sudo find /srv/salt/_tops -lname '/srv/salt/sd-*' -delete
```

Comment:
To get around the dangling symlink issue that you mention in your commit, couldn't we just do

    sudo find /srv/salt/_tops -type l -iname "sd*" -delete

or am I missing something obvious?

Reply:
@joshuathayer That'll prune all symlinks prefixed with `sd-*`, but the command in the commit message will only clean up _broken_ symlinks. It's _probably_ not necessary to do so, but wanted to document it in case someone a few months from now notices that the `clean-salt` target isn't robust enough.